tuist/tuist
# Expose the `.bundle` product type

Part of https://github.com/tuist/tuist/issues/290

### Short description 📝

Tuist currently supports iOS resource bundles however there isn't a way to leverage it directly from the project manifest.

### Solution 📦

- Expose the `.bundle` product type in the project manifest

### Implementation 👩‍💻👨‍💻

- [x] Expose `.bundle`

### Test Plan 🛠

- Run `tuist generate` within `fixtures/ios_app_with_framework_and_resources`
- Verify the project `StaticFramework` has a bundle target called `StaticFrameworkResources` which includes resources
- Verify the `StaticFrameworkResources` bundle is copied as a resouce in the App
- Verify running the app prints out descriptions for images within `StaticFrameworkResources`
